### PR TITLE
Documentation updated to reflect the changes made for Headless

### PR DESCRIPTION
**Pull request by sitereactor**
@hemraker @jmayntzhusen I tried to run through the Headless documentation and update to reflect the current state of the .net core client library. Do either of you two want to review these changes?

### CODE REVIEW

File: Umbraco-Cloud/Headless/Headless-Net-Client/Query.md [hunk 1]
```diff
@@ -11,30 +11,32 @@ To get content from Headless, you'll use the Headless Service described in the o
...
+    .EqualTo("Unicorn Pinot Noir");
+
 var contentWithSpecificName = 
-    await HeadlessClient
-    .Instance
-    .Query()
-    .Where(DefaultProperties.Name)
-    .IsEqualTo("Unicorn Pinot Noir")
-    .GetFirst();
+    await HeadlessClient.Instance.Query(query).First();
```
Comment (jmayntzhusen) [inline Umbraco-Cloud/Headless/Headless-Net-Client/Query.md]: I get an error on the .First() method: Task<PagedResult<ContentItem>> does not contain a definition for 'First'

File: Umbraco-Cloud/Headless/Headless-Net-Client/website-static.md [hunk 2]
```diff
@@ -22,47 +23,47 @@ _This example is for creating a statically routed website which means that it's
         }
         ```
 * You need to bootstrap the headless client which is done in your `Startup.cs` file:
-   * Add `using Umbraco.Headless.Client;`.
+   * Add `using Umbraco.Headless.Client.Net;`.
    * In `ConfigureServices` add the headless client services: `services.AddUmbracoHeadlessClient(Configuration);`
```
[the remark this answers is not in the document]
Comment (sitereactor) [reply Umbraco-Cloud/Headless/Headless-Net-Client/website-static.md]: Good catch! Looks like we are missing an extension method in the new client library.

I will get it added and let you know when its ready.

File: Umbraco-Cloud/Headless/Headless-Net-Client/console.md [hunk 3]
```diff
@@ -56,13 +54,13 @@ static void Main(string[] args)
         .BuildServiceProvider();
```
[the remark this answers is not in the document]
Comment (sitereactor) [reply Umbraco-Cloud/Headless/Headless-Net-Client/console.md]: Install this nuget package in your solution - it should be be a dependency on the .Web part of the library.: `Microsoft.Extensions.Configuration`

File: Umbraco-Cloud/Headless/Headless-Net-Client/Query.md (hunk 1 above)
Comment (sitereactor) [reply Umbraco-Cloud/Headless/Headless-Net-Client/Query.md]: yes, good catch. This needs to be divided into two parts.
First: `var result = await HeadlessClient.Instance.Query(query)` and then: `result.First()`.

File: Umbraco-Cloud/Headless/Headless-Net-Client/website-static.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sitereactor) [reply Umbraco-Cloud/Headless/Headless-Net-Client/website-static.md]: `UmbracoCms.Headless.Client 0.9.7-CI-20180903-01` and `UmbracoCms.Headless.Client.Web 0.9.7-CI-20180903-01` are now available on myget, which has this update.
The correct namespace for `AddUmbracoHeadlessClient` is `Umbraco.Headless.Client.Net.Web`

**Pull request merged**